Snowflake.Data/Core/FileTransfer/StorageClient/SFGCSClient.cs

```diff
@@ -504,12 +504,16 @@ private SFFileMetadata HandleFileHeaderErrForGeneratedUrls(WebException ex, SFFi
         /// <returns>File Metadata</returns>
         private SFFileMetadata HandleUploadFileErr(WebException ex, SFFileMetadata fileMetadata)
         {
-            Logger.Error("Failed to upload file: " + ex.Message);
+            Logger.Error("Failed to upload file. ", ex);
```

Comment:
Are you sure that the stack trace does not contain any sensitive data, e.g. encryption material?

Reply:
I'm not. I reverted this change.